package.json

```diff
@@ -70,6 +70,7 @@
     "@nuxtjs/opencollective": "^0.1.0",
     "@nuxtjs/youch": "^4.2.3",
     "babel-loader": "^8.0.0",
+    "babel-polyfill": "^6.26.0",
```

Comment:
There is already a @babel/polyfill in devDeps

Reply:
Whoops.